src/patternfly/demos/Page/page-template-header-tools-elements.hbs

```diff
@@ -1,29 +1,21 @@
 {{#> page-header-tools}}
-  {{#> toolbar}}
-    {{#> toolbar-group toolbar-group--modifier="pf-m-icons pf-screen-reader"}}
-      {{#> toolbar-item}}
-        {{#> button button--modifier="pf-m-plain" pagelayout-icon--bell="true" button--attribute='aria-label="Notifications"'}}
-          {{> button-icon button-icon--type="bell"}}
-        {{/button}}
-      {{/toolbar-item}}
-      {{#> toolbar-item}}
-        {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Settings"'}}
-          {{> button-icon button-icon--type="cog"}}
-        {{/button}}
-      {{/toolbar-item}}
-    {{/toolbar-group}}
-    {{#> toolbar-group}}
-      {{#> toolbar-item toolbar-item--modifier="pf-m-mobile"}}
-        {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Overflow actions"'}}
-          {{> button-icon button-icon--type="ellipsis-v"}}
-        {{/button}}
-      {{/toolbar-item}}
-      {{#> toolbar-item toolbar-item--modifier="pf-m-user pf-screen-reader"}}
+    <div class="pf-m-icons pf-screen-reader">
+      {{#> button button--modifier="pf-m-plain" pagelayout-icon--bell="true" button--attribute='aria-label="Notifications"'}}
```

Comment:
do you need `pagelayout-icon--bell="true"`

Reply:
nope!